alonet/torch2trt/quantization.py

```diff
@@ -81,6 +81,9 @@ def set_default_desc(self,  calib_method="histogram", per_channel_quantization=T
         ## convert Layers to QuantLayers
         quant_modules.initialize()
 
+        ## set fake quantization to True before torch2onnx
+        quant_nn.TensorQuantizer.use_fb_fake_quant = True
```

Comment:
Why do you have to do that ?

Reply:
This is mandatory, otherwise you can't call `torch.export.onnx` with a quantized model